WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -431,6 +436,36 @@ private extension StoreStatsAndTopPerformersViewController {
         stackView.addArrangedSubview(separator)
         return stackView
     }
+
+    func startCustomRangeTabCreation() {
+        guard let navigationController else { return }
+        customRangeCoordinator = CustomRangeTabCreationCoordinator(
+            navigationController: navigationController,
+            onDateRangeSelected: { [weak self] start, end in
+                self?.createCustomRangeTab(start, end)
+            }
+        )
+        customRangeCoordinator?.start()
+    }
+
+    func createCustomRangeTab(_ start: Date, _ end: Date) {
+        print(start.description + end.description)
+        let currentDate = Date()
+
+        // TODO: 11935 Do actual data fetching and displaying.
+        let customRangeVC = StoreStatsAndTopPerformersPeriodViewController(siteID: siteID,
+                                                                           timeRange: .custom(from: start, to: end),
+                                                                           currentDate: currentDate,
+                                                                           canDisplayInAppFeedbackCard: true,
+                                                                           usageTracksEventEmitter: usageTracksEventEmitter)
+        periodVCs.append(customRangeVC)
+
+        let customRangeTabbedItem = TabbedItem(title: "Custom Range",
+                                               viewController: customRangeVC,
+                                               accessibilityIdentifier: "todo-identifier")
```

Comment:
nit: Can we add a TODO with a GH issue to track this?

Reply:
This end up not needing an extra todo since I can now set the `accessibilityIdentifier` value (previously was waiting for the enum PR). Handled in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/3b752ee49b6eae5a496dabf0dd8bd2f40b177a8c